### PR TITLE
Find courses tab

### PR DESCRIPTION
**Pull request by aialopes**
Closes https://trello.com/c/kTOuwRgu/219-learning-find-courses-tab

Feature url: http://localhost:3000/account/learning/courses/

localhost_3000_account_learning_courses_

### CODE REVIEW

**Comment by rottitime** [comment]
@aialopes The learning 'goal' and 'distribution' are now components in:

- components/Account/LearningGoalBar/LearningGoalBar.tsx
- components/Account/LearningDistribution/LearningDistribution.tsx

File: web/pages/account/learning/courses.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import AccountLayout from '@/components/Layout/AccountLayout/AccountLayout'
+import Tabs from '@/components/UI/Tabs/Tabs'
+import { useEffect } from 'react'
+import FindCourses from '@/components/Account/FindCourses'
+import { useRouter } from 'next/router'
+import { SharedLearningHeader } from './index'
+
+const LearningRecordRedirect = () => {
```
Comment (rottitime) [inline web/pages/account/learning/courses.tsx:8]: `LearningRecordRedirect ` can be deleted. Tabs already have redirect attribute in place

File: web/pages/account/learning/courses.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <>
+      <SharedLearningHeader />
+
+      <Tabs
+        brandColor="brandLearning"
+        activeIndex={1}
+        tabItems={[
+          {
+            title: 'Learning record',
```
Comment (rottitime) [inline web/pages/account/learning/courses.tsx:34]: change to

```
            content: null,
            href: '/account/learning/'

File: web/pages/account/learning/index.tsx [hunk 3]
```diff
@@ -50,13 +61,30 @@ const LearningPage = () => {
...
       <Tabs
         brandColor="brandLearning"
         tabItems={[
           {
             title: 'Learning record',
             content: <LearningRecordList />
+          },
+          {
+            title: 'Find courses',
+            content: <FindCoursesRedirect />
```
Comment (rottitime) [inline web/pages/account/learning/index.tsx:87]: Replace with 

```
            content: null,
            href: '/account/learning/courses'

File: web/components/Account/FindCourses.tsx [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+                        course_type: e.target.value as Array<string>
+                      }
+                    },
+                    undefined,
+                    {
+                      scroll: false
+                    }
+                  )
+                }}
+                style={{
```
Comment (rottitime) [inline web/components/Account/FindCourses.tsx:370]: Can you move background-color to `/UI/Select/Select` as this needs to sit globally.

Also we already have a attribute for fullwidth. Delete `width` and `max-width` and add `fullWidth` to `Select`

```
              <Select
                fullWidth

**Review by rottitime — CHANGES_REQUESTED** [review]

File: web/package.json [hunk 5]
```diff
@@ -26,6 +26,7 @@
     "@mui/material": "^5.8.5",
     "@mui/x-data-grid": "^5.15.1",
     "@mui/x-date-pickers": "^5.0.1",
+    "date-fns": "^2.29.3",
```
[the remark this answers is not in the document]
Comment (aialopes) [reply web/package.json:30]: Yes. Had a check on their repo and it doesn't offer the same exact feature.

**Review by rottitime — APPROVED** [review]

**Pull request merged**